andela/coding-geeks-ah-frontend
# #168778797 finished setting up maintainability badge

#### What does this PR do?
It configures **Authors Haven** to use **code climate** for code maintainability.

#### Description of Task to be done?
1. Updating the README to display code climate code maintainability badge.

#### How should this be manually tested?
1. Check to see if you see the current code maintainability score on the code climate maintainability badge in the README.

#### Screenshots
<img width="681" alt="Screen Shot 2019-09-26 at 11 45 27" src="https://user-images.githubusercontent.com/34879554/65678197-2d64d100-e053-11e9-9ff6-37cbcd922aa7.png">
